db/comdb2.h

```diff
@@ -1665,7 +1665,7 @@ extern int gbl_lock_conflict_trace;
 extern int gbl_enque_flush_interval;
 extern int gbl_inflate_log;
 extern pthread_attr_t gbl_pthread_attr_detached;
-extern unsigned int gbl_nsql;
+extern uint32_t gbl_nsql;
```

Comment:
We need a convention/agreement on how to declare global unsigned int variables. Practically unsigned int is the same with uint32_t so we should do one or the other.